eslint/babel-eslint-parser/src/configuration.js

```diff
@@ -21,6 +21,26 @@ export function normalizeESLintConfig(options) {
   };
 }
 
+/**
+ * Merge the first user supplied estree plugin options to default estree plugin options
```

Comment:
Small thing, but this comment confused me at first because I read it as merging the first supplied plugin option instead of the first instance of the estree plugin. Maybe change to the following for clarity?
```suggestion
 * Merge user supplied estree plugin options to default estree plugin options
```

Reply:
Technically it's the first plugin descriptor whose `[0]` is `"estree"`, the others will be ignored anyway.

Reply:
Right, the description was just a bit confusing for me. But def not a blocker :)